### PR TITLE
Adding instructions for various hypervisors

### PR DESCRIPTION
**Pull request by atxbyea**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by fabaff — COMMENTED** [review]
We should move the whole content for a setup on a hypervisor to a separate page.

This page is already too long and we should keep the focus on the installation on physical hardware.

**Comment by frenck** [comment]
@fabaff Agreed, which will happen after all ADRs for the installation methods are approved (which will create focus). Maybe it is an acceptable solution to add it in the meantime? I think it is still valuable right now.

**Comment by atxbyea** [comment]
My formatting could probably have been better though

File: source/hassio/installation.markdown [hunk 1]
```diff
@@ -36,7 +36,11 @@ The following will take you through the steps required to install Home Assistant
...
+     - For VirtualBox create a new virtual machine, Select "Other Linux (64Bit) Assign it atleast 2GB of Memory and "Use an existing virtual hard disk file", select the VDI file from above, afterwards edit the "Settings" of the VM and go "System" then Motherboard and Enable EFI, then "Network" "Adapter 1" Bridged and your adapter
+     - For Hyper-V create a new virtual machine, Select "Generation 2", Assign it atleast 2GB of Memory and select "Connection -> "Your Virtual Switch that is bridged", then "Use an existing virtual hard disk" and select the VHDX file from above, after creation go to "Settings" -> "Security" and deselect "Enable Secure Boot"
+     - For KVM create a new virtual machine in virt-manager, select "Import existing disk image", provide the path to the QCOW2 image above, choose "Generic Default" for the OS, assign atleast 2GB RAM and 1vCPU, check the box for "Customize configuration before install" and select your bridge under "Network Selection", then under customization select "Overview" -> "Firmware" -> "UEFI x86_64: /usr/share/ovmf/x64/OVMF_CODE.fd"
```
Comment (cfergeau) [inline source/hassio/installation.markdown]: on my system (fedora 32, likely to be the same on EL systems), it's `/usr/share/edk2/ovmf/OVMF_CODE.fd`. It's prefilled, and the other UEFI option is "secboot" which I assume to be for secure boot.

Comment (fabaff) [reply source/hassio/installation.markdown]: On F32 with `virt-manager` 2.2.1 it's ` /usr/share/edk2/ovmf/OVMF_CODE.fd`. I can't remember where the `/usr/share/ovmf/x64/OVMF_CODE.fd` was coming from in the first place.

**Comment by fabaff** [comment]
Yes, we should add it.

File: source/hassio/installation.markdown (hunk 1 above)
Comment (cfergeau) [reply source/hassio/installation.markdown]: Debian/Ubuntu have `/usr/share/OVMF/OVMF_CODE.fd`, but no `/usr/share/ovmf/x64/OVMF_CODE.fd`

**Review by fabaff — APPROVED** [review]
I removed the path in the `virt-manager` section. `UEFI x86_64: ...` should be enough as at least with `virt-manager` 2.2.1 you can't edit the path in the UI.

**Pull request merged**